shared/src/store/action-reducers/exercise.ts

```diff
@@ -131,6 +135,10 @@ export namespace ExerciseActionReducers {
                     currentPatient.visibleStatusChanged
                 ) {
                     updateTreatments(draftState, currentPatient);
+                    logPatientVisibleStatusChanged(
+                        draftState,
+                        currentPatient.id
+                    );
```

Comment:
This might be a duplicate log entry, so you can remove it.

Reply:
i removed the other one (to assert dominance)